addons/spreadsheet/static/src/commands.json

```diff
@@ -333,24 +263,17 @@
           "y": 10
       },
       "definition": {
-          "metaData": {
-              "groupBy": [
-                  "stage_id",
-                  "country_id"
-              ],
-              "measure": "__count",
-              "order": "ASC",
-              "resModel": "crm.lead"
-          },
-          "searchParams": {
-              "comparison": null,
-              "context": {},
-              "domain": [],
-              "groupBy": [
-                  "stage_id",
-                  "country_id"
-              ],
-              "orderBy": []
+          "definition": {
+            "id": "bb8b5846-b950-4541-8bc0-8dbe30e14b63",
+            "groupBy": [
+              "stage_id",
+              "country_id"
+            ],
+            "measure": "__count",
+            "order": "ASC",
```

Comment:
I am not sure about that "order". The order is always applied on the measure.
Should it have the same shape as order clauses for pivots and lists ? even though the field is redundant.
`{ "field": "__count", "asc": true}`

Reply:
Yes I would go for the same shape of the others